glide.yaml.tmpl

```diff
@@ -7,9 +7,13 @@ import:
   - package: github.com/gogo/protobuf
     version: v1.0.0
 
+  # field_mask source
+  - package: github.com/google/protobuf
+
   # HTTP-gRPC reverse-proxy code generator
   - package: github.com/grpc-ecosystem/grpc-gateway
-    version: v1.3.1
+    repo:    https://github.com/infobloxopen/grpc-gateway.git
+    version: atlas-patch
```

Comment:
This atlas-patch is forked from master, it looked like, does that have any other consequences compared to v1.3.1?

Reply:
It should be forked from 1.4.1 (this is what is used everywhere). @askurydzin, please double check this.